### DISCUSSION TITLE
Oqtane.Survey Created

### DISCUSSION THREAD

**Discussion by ADefWebserver** [oqtane/oqtane.framework#1031] [Show and tell]
I created a new project:
https://github.com/oqtane/Oqtane.Survey

My desire is to implement the basic features of: [BlazorSimpleSurvey](https://github.com/ADefWebserver/BlazorSimpleSurvey)

* Unlimited Surveys
* Unlimited Survey Questions
* Survey responses in pie charts 
* Survey Question Types
  * Text Box
  * Text Area
  * Date
  * Date Time
  * Dropdown
  * Multi-Select Dropdown

I hope to complete this in the next two months.

**Comment by Line4GitHub** [comment]
How to let logged-in users modify their survey answers?

Thank you very much!

**Reply by ADefWebserver** [reply]
> How to let logged-in users modify their survey answers?

@Line4GitHub If logged in user takes the Survey again their answers will be updated. The module is at this location: https://github.com/oqtane/Oqtane.Survey

**Comment by Line4GitHub** [comment]
Thank you very much!

How to let logged-in users modify/edit (not re-enter) their survey answers?

Thank you very much!

**Reply by ADefWebserver** [reply]
> How to let logged-in users modify/edit (not re-enter) their survey answers?

You would have to download the source code and make a custom modification.

**Reply by Line4GitHub** [reply]
Thank you very much!
